captcha/events.py

```diff
@@ -60,3 +60,25 @@ async def on_member_join(self, member: Member):
     @commands.Cog.listener()
     async def on_member_remove(self, member: Member):
         await self.cleaner(member)
+
+    @commands.is_owner()
+    @commands.command()
+    async def captcha(self, ctx, *members: Member):
+        """Start a captcha challenge for the specified user"""
+        await ctx.send(
+            "The challenge has been launched for all members of the server specified in the command.\n"
+            + "```"
+            + ctx.message.content
+            + "```"
+        )
+        time = await self.data.guild(ctx.guild).timeout()
+        await self.data.guild(ctx.guild).timeout.set(20)
+        for i in members:
+            await self.runner(i)
+        await self.data.guild(ctx.guild).timeout.set(time)
+        await ctx.send(
+            "The challenge is over for all the users specified in the command. This does not mean that all users were not bots and therefore are still there.\n"
+            + "```"
+            + ctx.message.content
+            + "```"
+        )
```

Comment:
I am not very happy with how this message has been constructed. Adding together strings is slower, and unconventional for the most part. You should use F-strings instead. Also, adding message content on the end is unnecessary. See my suggestion below:

```suggestion
        message = (
            "**The challenge has finished for the following members**\n"
            + ", ".join(member.name for member in members if not member.bot)
        )
        if any(member.bot for member in members):
            message += (
                "\n\n**The following members were not challenged because they were bots**\n"
                + ", ".join(member.name for member in members if member.bot)
            )
        await ctx.send(message)
```